Model/src/main/java/org/apidb/apicommon/model/userdataset/ISASimpleTypeHandler.java

```diff
@@ -49,11 +50,12 @@ public String[] getInstallInAppDbCommand(UserDataset userDataset,
       metaObject.put("description", meta.getDescription());
       metaObject.put("summary", meta.getSummary());
       Files.write(metaJsonTmpFile, metaObject.toString().getBytes(StandardCharsets.UTF_8));
+      final String imageTag = Optional.ofNullable(System.getenv("ISA_IMAGE_TAG")).orElse("latest");
```

Comment:
I think we should check for an empty string here also.  I know in some cases we have a wrapping script set the environment with something like `export VALUE=${SOME_VALUE}` but if SOME_VALUE is missing, VALUE still gets set but to an empty string.